src/middlewared/middlewared/plugins/failover_/event.py

```diff
@@ -172,7 +172,7 @@ def run_call(self, method, *args):
             # `self.validate()` calls this method
             raise
         except Exception as e:
-            logger.error('Failed to run %s:%r:%s', method, args, e)
+            logger.error('Failed to run %s%r: %s', method, args, e)
```

Comment:
Since you've changed this would it make sense to remove `e` and just use `exc_info=True`?